addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1310,6 +1310,72 @@ registry.many2one = SnippetOption.extend({
     }
 });
 
+/**
+ * Handles the edition of snippet's anchor name.
+ */
+registry.anchor_name = SnippetOption.extend({
+    xmlDependencies: ['/web_editor/static/src/xml/editor.xml'],
+
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     */
+    openAnchorDialog: function (previewMode, value, $opt) {
+        var self = this;
+        this.anchorDialog = new Dialog(this, {
+            title: _t("Anchor Name"),
+            $content: $(qweb.render('web_editor.dialog.anchor_name', {
+                currentAnchor: self.$target.attr('id'),
+            })),
+            buttons: [
+                {
+                    text: _t("Validate"),
```

Comment:
Did this because of this in specs

> click on "Anchor Name", opens a modal with one input : Enter the name of the anchor Link. + validate or discard

In my opinion "Save" is also better